cmd/clusterctl/client/config.go

```diff
@@ -58,6 +58,37 @@ func (c *clusterctlClient) GetProviderComponents(provider string, providerType c
 	return components, nil
 }
 
+type ProcessYAMLOptions struct {
```

Comment:
In the future we can specify the following option if it is needed but for now this sub-command uses the `SimpleProcessor` default.
```
 // YamlProcessor defines the yaml processor to use for the cluster¬
 // template processing. If not defined, SimpleProcessor will be used.¬
 YamlProcessor Processor¬
```